src/classes/RD2_OpportunityEvaluation_BATCH.cls

```diff
@@ -58,6 +58,19 @@ public without sharing class RD2_OpportunityEvaluation_BATCH
         private set;
     }
 
+    /**
+    * @description Should Rollups be disabled before any new Installments Opps are inserted
+    */
+    public Boolean disableRollups {
```

Comment:
Does `disableRollups` have an effect if a user disables CRLP after enabling RD2?
Should the var be private?

Reply:
Is recently created [RD2_OpportunityEvaluationService.disableRollups()](https://github.com/SalesforceFoundation/Cumulus/blob/master/src/classes/RD2_OpportunityEvaluationService.cls#L178) method in PR #4789 obsolete now since rollup recalculation is disabled in a batch not in the service? 
Another difference is that current solution disables all rollups while the service disables only specific trigger handlers. Which one should be applied?

Reply:
Great catch. The `RD2_OpportunityEvaluationService.disableRollups()` method is definitely obsolete. They both did the same thing (disable trigger handlers related to rollups).

Also, if the User disables CRLP after enabling RD2 then rollups will still run, but it would be legacy rollups which isn't something we're disabling even in Legacy RD.